manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml

```diff
@@ -76,6 +76,10 @@ spec:
             periodSeconds: 30
           resources:
 {{ toYaml .Values.linux.resources | indent 12 }}
+          ports:
+            - containerPort: 8898
```

Comment:
Do you intend to make it configurable? If so can we use the value from values.yaml file? We can use the same default value.

We also need to refer to the same here - 
https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/d40addbcc57a87a61f898573f73a8fad9dc6621f/manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer.yaml#L28


https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/d40addbcc57a87a61f898573f73a8fad9dc6621f/manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer-windows.yaml#L28

Reply:
done